src/linear_algebra/dual.lean

```diff
@@ -886,6 +896,38 @@ begin
   exact finrank_range_add_finrank_ker f,
 end
 
+open linear_map
+/-- `f.dual_map` is injective if and only if `f` is surjective -/
+@[simp] lemma dual_map_injective_iff
+  (f : V₁ →ₗ[K] V₂) :
+  function.injective f.dual_map ↔ function.surjective f :=
+begin
+  refine ⟨_, λ h, dual_map_injective_of_surjective h⟩,
+  rw [← range_eq_top, ← ker_eq_bot],
+  intro h,
+  apply finite_dimensional.eq_top_of_finrank_eq,
+  rw ← finrank_eq_zero at h,
+  rw [← add_zero (finite_dimensional.finrank K f.range), ← h,
+      ← linear_map.finrank_range_dual_map_eq_finrank_range,
+      linear_map.finrank_range_add_finrank_ker, subspace.dual_finrank_eq],
+end
+
+/-- `f` is injective if and only if `f.dual_map` is surjective -/
+@[simp] lemma dual_map_surjective_iff [finite_dimensional K V₁] (f : V₁ →ₗ[K] V₂) :
+  function.injective f ↔ function.surjective f.dual_map :=
```

Comment:
I commented this already but I think it got lost; please flip the direction of the statement and the comment. The name is correct already.

Reply:
fixed